server/src/models/Vote.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const Model = require("./Model")
+
+class Vote extends Model {
+  static get tableName() {
+    return "votes"
+  }
+
+  static get relationMappings() {
+    const { Podcast, User } = require('./index.js')
+    return {
+      users: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: User,
+        join: {
+          from: "votes.userId",
+          to: "users.id"
+        }
+      },
+      podcasts: {
+        relation: Model.HasManyRelation,
+        modelClass: Podcast,
+        join: {
+          from: "votes.podcastId",
+          to: "podcasts.id"
+        }
```

Comment:
I see your intention with this relation, but am concerned that you aren't thinking of it consciously
- a vote belongs to one podcast, you cant vote for two podcasts in one stroke. the relation should be a `BelongstoOne`
- `podcasts` should be `podcast` due to the BelongsToOne` relation